### PR TITLE
update fortISSimO.asm

### PR DESCRIPTION
**Pull request by MrBlinky**
optimize hUGE_SelectSong

### CODE REVIEW

File: fortISSimO.asm [hunk 1]
```diff
@@ -117,35 +117,31 @@ hUGE_SelectSong::
...
+	ld [hl], c
 .ch1NotOurs
-	rr c
+	rra
 	jr c, .ch2NotOurs
-	xor a
-	ldh [rNR22], a
-	ld a, AUDENV_UP
-	ldh [rNR22], a
+	ld l,rNR22 & $FF
```
Comment (ISSOtm) [inline fortISSimO.asm:129]:
```suggestion
	ld l, LOW(rNR22)
```

File: fortISSimO.asm [hunk 2]
```diff
@@ -117,35 +117,31 @@ hUGE_SelectSong::
...
+	ld [hl], b
+	ld [hl], c
 .ch2NotOurs
-	rr c
+	rra
 	jr c, .ch3NotOurs
-	ld hl, rNR30
+	ld l, rNR30 & $FF
```
Comment (ISSOtm) [inline fortISSimO.asm:135]:
```suggestion
	ld l, LOW(rNR30)
```

File: fortISSimO.asm [hunk 3]
```diff
@@ -117,35 +117,31 @@ hUGE_SelectSong::
...
 	ld [hl], h ; This has bit 7 set.
 .ch3NotOurs
-	rr c
+	rra
 	jr c, .ch4NotOurs
-	xor a
-	ldh [rNR42], a
-	ld a, AUDENV_UP
-	ldh [rNR42], a
+	ld l,rNR42 & $FF
```
Comment (ISSOtm) [inline fortISSimO.asm:141]:
```suggestion
	ld l, LOW(rNR42)
```

**Review by ISSOtm — CHANGES_REQUESTED** [review]
Thank you! I would just like a small stylistic change.

**Comment by MrBlinky** [comment]
Looks much better. Updated

**Review by ISSOtm — APPROVED** [review]
Thank you!

**Pull request merged**